rc/extra/racer.kak

```diff
@@ -126,12 +133,13 @@ define-command racer-show-doc -docstring "Show the documentation about the rust
     %sh{
         dir=${kak_opt_racer_tmp_dir}
         cursor="${kak_cursor_line} ${kak_cursor_column}"
-        racer_data=$(racer --interface tab-text  complete-with-snippet  ${cursor} "${kak_buffile}" "${dir}/buf" | head -n 2 | tail -n 1)
+        racer_data=$(racer --interface tab-text  complete-with-snippet  ${cursor} "${kak_buffile}" "${dir}/buf" | sed -n 2p )
         racer_match=$(printf %s\\n "$racer_data" | cut -f1)
         if [ "$racer_match" = "MATCH" ]; then
           racer_doc=$(printf %s\\n "$racer_data" | cut -f9 )
-          racer_doc=$(printf %s\\n "$racer_doc" | sed -e 's/^"\(.*\)"$/\1/g')
-          racer_doc=$(printf %s\\n "$racer_doc" | sed -e "s/@/\\\\@/g")
+          racer_doc=$(printf %s\\n "$racer_doc" |
+            sed -e 's/^"\(.*\)"$/\1/g' | # Remove leading and trailing quotes
+            sed -e "s/@/\\\\@/g")        # Escape all @ so that it can be properly used in the string expansion
```

Comment:
I think the idea was to have something like the following, to avoid spawning an extra process:

```
racer_doc=$(printf %s\\n "$racer_doc" |
       sed -e 's/^"\(.*\)"$/\1/g' \
           -e "s/@/\\\\@/g")
```